cirq-core/cirq/circuits/qasm_output.py

```diff
@@ -293,7 +293,12 @@ def output(text):
                 output(f'creg {meas_id}[{len(meas.qubits)}];\n')
             else:
                 output(f'creg {meas_id}[{len(meas.qubits)}];  // Measurement: {comment}\n')
-        output_line_gap(2)
+        # In OpenQASM 2.0, the transformation of global phase gates is ignored.
+        # Therefore, no newline is created when the operations contained in a circuit consist only of global phase gates.
+        if not (
```

Comment:
change the condition so that it works for the when the circuit has multiple global phases
```py3
if all(isinstance(op, ops.GlobalPhaseGate) for op in self.operations):
```

Reply:
I overlooked that pattern. I've addressed it now. 
Thank you for pointing it out!